packages/preview/vercanard/1.0.0/template.typ

```diff
@@ -0,0 +1,62 @@
+#let entry(title, body, details) = [
+    #heading(level: 2, title)
+    #body
+
+    #text(fill: gray, details)
+]
+
+#let resume(name: "", title: "", accent_color: rgb("db9df8"), margin: 100pt, aside: [], body) = {
```

Comment:
I would recommend `kebab-case` as the idiomatic Typst case, but it's ultimately up to you.